src/kata5.reachDestination.js

```diff
@@ -1,5 +1,26 @@
 const reachDestination = (distance, speed) => {
 
+  function reachDestination (distance, speed) {
```

Comment:
Not sure why you have a function within a function here - you definitely don't need it!